src/components/Article.jsx

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import Header from "./Header";
+import Navbar from "./Navbar";
+
+export default function Article({ articleId }) {
+  const [article, setArticle] = useState();
+
+  useEffect(() => {
+    axios
+      .get(`https://my-web-service-dwf8.onrender.com/api/articles/${articleId}`)
+      .then((response) => {
+        setArticle(response.data.article[0]);
+      })
+      .catch((err) => {
+        console.log("there's an err", err);
+      });
+  }, []);
+
+  return (
+    <>
+      <Header></Header>
```

Comment:
if header and nav bar are going to be in all your components you can render them in app